src/components/pages/home/index.tsx

```diff
@@ -216,72 +217,74 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
+                            })
+                          }}
+                        >
+                          <Image
+                            quality={100}
+                            src={
+                              'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1617475003/egghead-next-pages/home-page/eggo-gardening.png'
+                            }
+                            width={250}
+                            height={305}
+                            alt={featureDigitalGardening.title}
+                          />
+                        </a>
+                      </Link>
                     </div>
-                    <div className="grid md:grid-cols-12 grid-cols-2 gap-5 mt-12">
-                      {featureDigitalGardening.featured.courses.map(
-                        (resource: any) => {
-                          return (
-                            <VerticalResourceCard
-                              className="col-span-4 text-center border border-gray-200"
-                              key={resource.path}
-                              resource={resource}
-                              location={location}
-                            />
-                          )
-                        },
-                      )}
+                    <div className="sm:col-span-2 flex flex-col sm:items-start items-center w-full">
+                      <h3 className="text-xs text-green-700 dark:text-green-400 uppercase font-semibold mb-2">
+                        Learn in public with a digital garden
+                      </h3>
+                      <Link href={featureDigitalGardening.path}>
+                        <a
+                          className="font-bold hover:text-blue-600 dark:hover:text-blue-300 transition ease-in-out"
+                          onClick={() => {
+                            track('clicked resource', {
+                              resource: featureDigitalGardening.path,
+                              location,
+                            })
+                          }}
+                        >
+                          <h2 className="sm:text-2xl md:text-4xl text-xl max-w-screen-lg font-extrabold leading-tighter">
+                            {featureDigitalGardening.title}
+                          </h2>
+                        </a>
+                      </Link>
+                      <div>
+                        <Markdown className="prose dark:prose-dark dark:prose-sm-dark mt-4">
+                          {featureDigitalGardening.description}
+                        </Markdown>
+                        <Markdown className="prose dark:prose-dark dark:prose-sm-dark mt-4 font-medium">
+                          {featureDigitalGardening.quote.description}
+                        </Markdown>
+                      </div>
                     </div>
                   </div>
+                  <div className="grid grid-cols-3 gap-5 mt-12">
```

Comment:
moved the 3 card to the bottom of the digital garden section so it could take up the full row instead of just the width of the description. Cards were acting weird because they didn't have enough room